Firmware/MotorControl/motor.cpp

```diff
@@ -108,9 +116,9 @@ bool Motor::check_DRV_fault() {
         // Update DRV Fault Code
         drv_fault_ = DRV8301_getFaultType(&gate_driver_);
         // Update/Cache all SPI device registers
-        DRV_SPI_8301_Vars_t* local_regs = &gate_driver_regs_;
-        local_regs->RcvCmd = true;
-        DRV8301_readData(&gate_driver_, local_regs);
+        // DRV_SPI_8301_Vars_t* local_regs = &gate_driver_regs_;
```

Comment:
was this commented out just for testing? (same in protocol exports)

Reply:
as per discussion, this is on purpose to make the fault register fetch as fast as possible